debugger/gdb-sgx-plugin/mman.py

```diff
@@ -36,10 +36,10 @@ class myst_vad_t:
     OFFSETOF_SIZE = 24
     SIZEOF_SIZE = 8
 
-    OFFSETOF_PROT = 30
+    OFFSETOF_PROT = 32
```

Comment:
We also need to lockdown that this value matches the C source code.

Here is one strategy of locking down:
https://github.com/openenclave/openenclave/search?q=ENCLAVE_OFFSETOF_PATH